lib/libc/minimal/include/inttypes.h

```diff
@@ -46,4 +46,12 @@
 #define	PRIX64			"llX"		/* uint64_t */
 #define	PRIXPTR			"lX"		/* uintptr_t */
 
+#if SIZE_MAX == UINT32_MAX
+#define LOCAL_PRIxOFF		"08zx"		/* off_t */
+#elif SIZE_MAX == UIN64_MAX
+#define LOCAL_PRIxOFF		"16zx"		/* off_t */
+#else
+#error Formatting undefined for this size of ssize_t/off_t
+#endif
```

Comment:
It can not be done in this file, because these format strings are somehow standard and it does not seem that `PRIxOFF` exists. You can not define `LOCAL_` things here.
Additional problem is that SIZE_MAX does not seem to be defined as these two values only and the `inttypes.h` is included into many places where this triggers compiler errors.
If you want to add the `LOCAL_PRIxOFF` do it at the source files you are using it, but add some `TODO` above the definition so that the define would be removed when it finally gets proper `inttypes.h` definition.

Reply:
Just do not touch the `inttypes.h`.

Reply:
Right, let's put this off for a while since I'm not in favor of having a separate specifier per each module where it's being used.